ci-scripts/osx/travis-install.sh

```diff
@@ -6,11 +6,3 @@ brew install clang-format
 # from Homebrew 1.6.0 the old formula for obtaining Qt5.9.2 becomes invalid.
 # so we start to use the latest version of Qt. (#1910)
 brew install qt
-# temp workaround to brew installed glew cmake info overriding glew lib detection
-# which causes compiling issues later
-if [ -L /usr/local/lib/cmake/glew ]
```

Comment:
Is there a reason you are removing this? To my knowledge, this is still needed for OSX Travis builds to build successfully.